sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-bitmap-from-image.js

```diff
@@ -58,12 +60,18 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
+        var testCanvas = document.createElement('canvas');
+        var ctx = testCanvas.getContext("2d");
+        setCanvasToRedGreen(ctx);
```

Comment:
No problem. I tried GIMP before, that didn't work. But I managed to generate a .png file with half transparency using Matlab. I will change this in a follow-up pull request. Thank you.